src/ostree/ot-remote-builtin-add.c

```diff
@@ -51,6 +52,7 @@ static GOptionEntry option_entries[] = {
   { "if-not-exists", 0, 0, G_OPTION_ARG_NONE, &opt_if_not_exists, "Do nothing if the provided remote exists", NULL },
   { "force", 0, 0, G_OPTION_ARG_NONE, &opt_force, "Replace the provided remote if it exists", NULL },
   { "gpg-import", 0, 0, G_OPTION_ARG_FILENAME, &opt_gpg_import, "Import GPG key from FILE", "FILE" },
+  { "custom-backend", 0, 0, G_OPTION_ARG_STRING, &opt_custom_backend, "This remote has content not fetched via libostree", "NAME" },
```

Comment:
Is `--contenturl` here below compatible with this?
In other word, is a custom-backed handler meant to handle a `(Option<Url>, Option<ContentUrl>)` for a remote?

Reply:
It's allowed to set `url` in addition; in fact I think I'm going to rework ostree-rs-ext to require that for container fetches.

Updated the test suite.

For `--contenturl`...I don't see a reason to disallow it explicitly.  Maybe it makes sense for the backend.  If it doesn't, don't set it.